content/de/events/2021-03/talk-energy-consumption

```diff
@@ -0,0 +1,20 @@
+---
+title: "Behavioural insights to change energy consumption patterns in Kosovo"
+subtitle: "Presented by CorrelAidX Cologne"
+date: 2020-03-11T13:01:56+02:00
```

Comment:
the event does not show up on the [deploy preview](https://deploy-preview-184--correlaid-website-preview.netlify.app/events/) but this could be due to the "date" slot being in UTC and not in UTC+1 (so hugo doesn't think this should be published already).